pkiintergration-fe/src/services/create-certificate.service.ts

```diff
@@ -1,29 +1,38 @@
 import { Injectable } from '@angular/core';
 import { FetchClient } from '@c8y/client';
-import { AlertService, gettext } from '@c8y/ngx-components';
+import { AlertService } from '@c8y/ngx-components';
 
 @Injectable({
   providedIn: 'root'
 })
 export class CreateKeyPairService {
   constructor(private fetchClient: FetchClient, private alertService: AlertService) {}
+
   async getKeyPair(deviceId: string): Promise<void> {
     try {
-      //Dummy service, there are not available from Backend
-      const response = await this.fetchClient.fetch('createOperationForDevice/{deviceId}', {
-        method: 'POST',
-        body: JSON.stringify({ deviceId }),
-        headers: { 'Content-Type': 'application/json' }
-      } as RequestInit);
-      // TODO: Remove console
+      const response = await this.fetchClient.fetch(
+        `/service/pki-integration/operations/createOperationForDevice/${deviceId}`,
+        {
+          method: 'GET',
+          // body: JSON.stringify({ deviceId }),
+          // headers: { 
+          //   'Content-Type': 'application/json',
+          //   Accept: 'application/json'
+          // }
+        }
+      );
+
       console.log('res: ', response);
+
       if (response.status >= 200 && response.status < 300) {
-        this.alertService.success(gettext('Operation for Device created'));
+        const body = await response.json();
+        console.log(body);
```

Comment:
todo: mark as "work in progress" or resolve (console, comment)

Reply:
reslove because end of project